schema/data/prod/form_json/administration.json

```diff
@@ -2,8 +2,9 @@
   "schema": {
     "type": "object",
     "properties": {
-      "comments": {
-        "type":"string"
+      "adminComments": {
+        "type":"string",
+        "title": "comments"
```

Comment:
Rule: Each id must be unique on the page.

The property names are used by react-jsonschema-form to set ids on some of the corresponding form elements (prefixed by `root_`). Because the comments property was called "comments" for each form tab (administration, emission, fuel, production) this was causing it to create many duplicate `root_comments` ids.

Reply:
This is a breaking change, as it would cause comments from already submitted applications to not be rendered anymore. To fix that a11y issue, you should use the idPrefix prop instead: https://react-jsonschema-form.readthedocs.io/en/v1.8.1/advanced-customization/#id-prefix